docs/source/algorithms/SofTwoThetaTOF-v1.rst

```diff
@@ -36,8 +36,8 @@ Usage
    from mantid.simpleapi import *
    import directtools as dt
    ws = CreateSampleWorkspace(Function='Quasielastic Tunnelling', NumBanks=1, BankPixelWidth=20)
-   AddSampleLog(ws, 'wavelength', '6.26', 'Number', 'Angstrom')
-   SetInstrumentParameter(ws, ParameterName='l2', ParameterType='String', Value='5')
+   AddSampleLog(ws, LogName='wavelength', LogValue='6.26', LogType='Number', LogUnit='Angstrom')
```

Comment:
```suggestion
   AddSampleLog(ws, LogName='wavelength', LogValue=6.26, LogType='Number', LogUnit='Angstrom')
```
I am not sure this will fix the sphix docs error, perhaps it can do something similar to [this above?](https://github.com/mantidproject/mantid/pull/25601/files#diff-e67b4aea70a709194bbdaaf884e95e24R33)